src/components/DowTriagePoc.vue

```diff
@@ -0,0 +1,274 @@
+<template>
+  <div>
+    <h1>DOW Triage POC</h1>
+    <div class="container">
+
+      <div id="q1-row" class="question-row">
+        <div class="question">
+          <label for="q1">Are you building over a pitch fiber pipe, asbestos, cement, brick?</label>
+          </div>
+        <div class="answer">
+          <input type="radio" name="q1" v-bind:value="true" id="q1-yes" v-model="vq1"  >
```

Comment:
On all these ones, you don't need to bind the value of it here, you can set the value of vq1 in the vue file

Reply:
I think here though if you just use "value=true" for example then it makes the value a string, as opposed to bool.  However by binding it, it allows it to use bool (making the logic in the script a bit cleaner).  In the main DwrCymru code, radios are normally string values (eg "Yourself", "A third party") and I didn't want to use "true" as a string (or "yes/no") when I could use the bool values in the code.  Does that make sense, would you have approached a different way (given this is largely a disposable POC demo)?  Cheers.